pkg/crc/segment/segment.go

```diff
@@ -0,0 +1,74 @@
+package segment
+
+import (
+	"io/ioutil"
+	"os"
+
+	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/code-ready/crc/pkg/crc/logging"
+	"github.com/code-ready/crc/pkg/crc/version"
+	"github.com/pborman/uuid"
+	"github.com/segmentio/analytics-go"
```

Comment:
Should this be versioned? `import "gopkg.in/segmentio/analytics-go.v3"`

Reply:
There is issue with the library right now https://github.com/segmentio/analytics-go/issues/170 which doesn't behave correctly with go-1.15